onadata/libs/tests/serializers/test_submission_review_serializer.py

```diff
@@ -0,0 +1,107 @@
+# -*- coding: utf-8 -*-
+"""
+Submission Review Serializer Test Module
+"""
+from __future__ import unicode_literals
+
+from rest_framework.exceptions import ValidationError
+
+from onadata.apps.logger.models import Instance, Note, SubmissionReview
+from onadata.apps.main.tests.test_base import TestBase
+from onadata.libs.serializers.submission_review_serializer import \
+    SubmissionReviewSerializer
+
+
+class TestSubmissionReviewSerializer(TestBase):
+    """
+    TestSubmissionReviewSerializer Class
+    """
+
+    def _create_submission_review(self):
+        """
+        Utility to create a submission review
+        """
+
+        self._publish_transportation_form_and_submit_instance()
+
+        instance = Instance.objects.first()
+
+        data = {
+            "instance": instance.id,
+            "note_text": "Hey there",
+            "status": SubmissionReview.APPROVED
+        }
+
+        serializer_instance = SubmissionReviewSerializer(data=data)
+        self.assertFalse(Note.objects.filter(instance=instance).exists())
+
+        self.assertTrue(serializer_instance.is_valid())
+
+        submission_review = serializer_instance.save()
+
+        # Creates Note Object
+        self.assertTrue(Note.objects.filter(instance=instance).exists())
+
+        note = submission_review.note
+        self.assertEqual(instance, submission_review.instance)
```

Comment:
it would be cool to have this more consistent.  Maybe expected values on the left and received values on the right?

Reply:
Updated the tests to be more consistent in the latest commits.